changelog.d/257.bugfix

```diff
@@ -0,0 +1 @@
+Fix .toUpperCase() errors
```

Comment:
uh.

Reply:
This is actually the user facing error, so it's fine to put this into the changelog (although I should probably have made this more detailed).